actions/castVote.ts

```diff
@@ -50,9 +50,11 @@ export async function castVote(
   proposal: ProgramAccount<Proposal>,
   tokenOwnerRecord: ProgramAccount<TokenOwnerRecord>,
   voteKind: VoteKind,
+  isMulti: boolean,
```

Comment:
this parameter is redundant and should be removed. we already know if it's a multi vote just by checking `proposal.account.voteType`